features/networking/ovn-egress-ip.feature

```diff
@@ -482,3 +482,151 @@ Feature: OVN Egress IP related features
       | curl | -s | --connect-timeout | 5 | <%= cb.ipecho_url %> |
     Then the step should succeed
     """
+
+  # @author huirwang@redhat.com
+  # @case_id OCP-33718
+  @admin
+  @destructive
+  Scenario: Deleting EgressIP object and recreaing it will work
+    Given I save ipecho url to the clipboard
+
+    #Get unused IP as egress ip
+    Given I store a random unused IP address from the reserved range to the clipboard
+
+    #Create a project and pods in it,add label to the namespace
+    Given I have a project
+    And evaluation of `project.name` is stored in the :proj1 clipboard
+    When I run the :label admin command with:
+      | resource | namespace           |
+      | name     | <%= cb.proj1 %>     |
+      | key_val  | og=qe               |
```

Comment:
typo: `s/og/org/`?

Reply:
hmmm, it is just label name, no special meaning... I think it is ok to leave it as it is.

Reply:
@huiran0826 actually word `og` is a name of one of the Red Hat Employees :) I'd prefer if you could change it to `org`.

Reply:
Thanks for your comments. Actually If I update the "og" to "org" here, there are many places need to update...

Reply:
@huiran0826 I'll merge this and file another PR tomorrow to fix the label issue.  
```
$ rg 'og=qe'
features/networking/ovn-egress-ip.feature
21:      | key_val  | og=qe               |
85:      | key_val  | og=qe               |
150:      | key_val  | og=qe               |
160:      | key_val  | og=qe               |
190:      | key_val  | og=qe               |
218:      | key_val  | og=qe               |
268:      | key_val  | og=qe               |
325:      | key_val  | og=qe               |
371:      | key_val  | og=qe               |
456:      | key_val  | og=qe               |
```

Reply:
@pruan-rht Thanks very much for taking care of it. Did you already open the PR, if not, let me open it, not only this file, but also testdata file.

Reply:
Filed PR https://github.com/openshift/verification-tests/pull/1873

Reply:
@huiran0826 I just did filed a PR and then I saw this message. :). Thanks for the quick turnaround...merged that new PR already.